libnetwork/bitmap/sequence.go

```diff
@@ -481,14 +484,16 @@ func findSequence(head *sequence, bytePos uint64) (*sequence, *sequence, uint64,
 // A) block is first in current:         [prev seq] [new] [modified current seq] [next seq]
 // B) block is last in current:          [prev seq] [modified current seq] [new] [next seq]
 // C) block is in the middle of current: [prev seq] [curr pre] [new] [curr post] [next seq]
-func pushReservation(bytePos, bitPos uint64, head *sequence, release bool) *sequence {
+//
+// The bool return is true if the bit value was changed.
+func pushReservation(bytePos, bitPos uint64, head *sequence, release bool) (*sequence, bool) {
```

Comment:
This would be a case where possibly a named output variable could be OK;


```suggestion
// The bool return is true if the bit value was changed.
func pushReservation(bytePos, bitPos uint64, head *sequence, release bool) (_ *sequence, changed bool) {
```

or

```suggestion
// The bool return is true if the bit value was changed.
func pushReservation(bytePos, bitPos uint64, head *sequence, release bool) (newHead *sequence, changed bool) {
```

Reply:
Done.

Reply:
Thanks!